src/main/java/anichan/bookmark/Bookmark.java

```diff
@@ -74,6 +74,11 @@ public void addNote(int bookmarkIndex, String note) {
         this.noteList.get(bookmarkIndex).addNote(note);
     }
 
+    public String removeNote(int bookmarkIndex, int noteIndex) {
+        String removeNote = noteList.get(bookmarkIndex).removeNote(noteIndex);
```

Comment:
Perhaps you want to avoid combining 2 method calls in 1 line?

Reply:
Ok noted, I will make changes to it.